docs/java/index.md

```diff
@@ -12,20 +12,20 @@ sidebar: auto
 sidebarDepth: 0
 
 ---
-# Ethereum for Java Developers
+# Ethereum ♡ Java
 
 <div class="featured">Learn how to develop for Ethereum using Java-based projects and tooling</div><br>
 
 Use Ethereum to create decentralized applications (or “dapps”) that utilize the benefits of cryptocurrency and blockchain technology. These dapps can be trustworthy, meaning that once they are deployed to Ethereum, they will always run as programmed. They can control digital assets in order to create new kinds of financial applications. They can be decentralized, meaning that no single entity or person controls them and are nearly impossible to censor.
 
-<img src="https://asdgvdoyen.cloudimg.io/width/2560/webp/https://api.kauri.io:443/ipfs/QmPyoXQaK9uA1oedsptssr1EhYRBF1A9vrnypbKAkMhuxQ" width="100%" />
+![Ethereum-Java](https://asdgvdoyen.cloudimg.io/width/2560/webp/https://api.kauri.io:443/ipfs/QmPyoXQaK9uA1oedsptssr1EhYRBF1A9vrnypbKAkMhuxQ)
```

Comment:
Let's keep this as is... this is what it looks like without the `width="100%"`:
<img width="1438" alt="Image 2019-09-19 at 9 05 25 PM" src="https://user-images.githubusercontent.com/8097623/65273360-46382880-db21-11e9-95ba-4e2999cf8185.png">
FYI your PRs will create a preview deploy in Netlify where you can see how it looks:
https://deploy-preview-246--ethereumorg.netlify.com/java/

Reply:
Preview is great for when I'm not working on my daily station with `vuepress` preview.